src/main/java/com/karankumar/bookproject/backend/statistics/PageStatistics.java

```diff
@@ -67,6 +67,6 @@ public Integer calculateAveragePageLength() {
             return null;
         }
         return (booksWithPagesSpecified == 0) ? 0 :
-                (int) Math.ceil(totalNumberOfPages / booksWithPagesSpecified);
+                (int) Math.ceil(totalNumberOfPages / (float) booksWithPagesSpecified);
```

Comment:
Could you please remove the `int` cast? I think it's better to return float here (sorry, I should have mentioned this in the issue)